test/thermodynamics/isobaric_reaction_potential_test.py

```diff
@@ -0,0 +1,59 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (assert_equal, units, Quantity, errors)
+from symplyphysics.laws.thermodynamics import isobaric_reaction_potential as potential_law
+
+# Description
+## The decomposition of barium carbonate by the reaction: BaCO3 = BaO + CO2.
+## At a temperature of 298 kelvin, the thermal effect of the reaction is 552866 joule per mole, the entropy change is 37.22 [joule / (mole * kelvin)].
+## Then the isobaric potential is 506676 joule per mole.
+## https://studfile.net/preview/5797036/page:4/
```

Comment:
Same issues as in another PR.

Reply:
Yes, another reaction is considered there, but it shows where the values come from. There are contradictions in tabular values for the reaction by reference, so I considered another reaction. For another reaction, I have the exact values, but no link.